Dockerfile

```diff
@@ -26,3 +26,8 @@ COPY *.cfg *.conf *.sh *.in *.txt *.py ${PROJECT_HOME}/
 
 # tests
 COPY .pylintrc .flake8 ${PROJECT_HOME}/
+
+# labels
+ARG commit
```

Comment:
`IMAGE_TAG` should always be a commit if passed by Jenkins, whereas local images should never be pushed (the local default is for them)

Reply:
That may be the case, but the name doesn't give that away. I wouldn't blame anyone for changing it, not knowing that it is expected to be the commit somewhere. I can use `IMAGE_TAG` if you really think that this is what we should do.

Reply:
I understand the `ARG` should be named `commit` now, as why would you have a label with the value of a tag? You can have multiple tags on the same Docker image so it is ill-defined. That said, `docker-compose.yml` can pass in
```
args: 
    commit: ${IMAGE_TAG}
```
so that it gets the value from Jenkins conventions. In retrospect it could have been named more fluently but its primary usage is indeed being the image tag.

Reply:
Okay, I will pass it in as commit. Sometimes it's okay to have two variables, even if most of the time they are the same. I just ask to get it right, because I am planning to apply it to other projects in the same way (basically every Docker image should then have those labels populated - not that I am planning to go through all projects now).

Reply:
That's changed now